src/components/PostPageDetail/index.js

```diff
@@ -44,7 +44,6 @@ import useWriteComment from '../Comments/hooks/useWriteComment';
 import LoadingWithoutModal from '../LoadingWithoutModal';
 import {Shimmer} from '../Shimmer/Shimmer';
 import Content from './elements/Content';
-import usePostDetail from './hooks/usePostDetail';
 import usePostHook from '../../hooks/core/post/usePostHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `usePostDetail` hook has been replaced with `usePostHook`. Make sure that all the functionalities provided by `usePostDetail` are covered by `usePostHook` or other parts of the code.
```diff
- import usePostDetail from './hooks/usePostDetail';
+ import usePostHook from '../../hooks/core/post/usePostHook';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->